rllib/algorithms/ppo/tests/test_ppo_rl_module.py

```diff
@@ -47,7 +47,7 @@ def get_expected_module_config(
     return config
 
 
-def dummy_torch_ppo_loss(batch, fwd_out):
+def dummy_torch_ppo_loss(module, batch, fwd_out):
```

Comment:
did we not get rid of this at some point?

Reply:
I can't recall us getting rid of it and it's still on master.